data-raw/process_data_set_jessica.R

```diff
@@ -0,0 +1,18 @@
+library(tidyverse)
+library(stringr)
+library(lubridate)
+library(janitor)
+library(usethis)
+
+nba_all_elo <- read_csv("data-raw/nba-elo/nbaallelo.csv")%>%
+  arrange(game_id, `_iscopy`)%>%
```

Comment:
For future reference, running `clean_names()` from the above loaded `janitor` package after line 7 will clean up all tricky variable names like `_iscopy` for you automagically